README.md

```diff
@@ -4,10 +4,24 @@
 
 LXImage-Qt is the Qt port of LXImage, a simple and fast image viewer.
 
-In addition it features a tool to take screenshots.
+![LXImage-qt](lximage-qt.png)
 
-It is maintained by the LXQt project but can be used independently from this
-desktop environment.
+
+## Features
+
+* Zoom, rotate, flip and resize images
+* Slideshow
+* Thumbnail bar (left, top or bottom); different thumbnail sizes
+* Exif data bar
+* Inline image renaming
+* Custom shortcuts
+* Image annotations (arrow, rectangle, circle, numbers)
+* Recent files
+* Upload images (Imgur, ImgBB)
+* Take screenshots
+
+Even more features that can be found using it. LXImage-qt is maintained by the LXQt project
```

Comment:
The English has a small problem: "...that can be found using it" → "...that can be found when it is used".

Reply:
Maybe more than one problem...

`More features can be found when it is used.` ?

Reply:
> More features can be found when it is used.

Yes. It's better.

Reply:
BTW, some people might say it's "LXImage-Qt", not "LXImage-qt".

Reply:
We are the DE with most use of shift key definitevely :)
Will change it.